{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/css/base/define.css

```diff
@@ -7,6 +7,26 @@
 | Global resets to make life easier
 |
 */
+a,
```

Comment:
I think explaining this part of the file in a comment would be good - the other resets in the file have one.

Reply:
Sure thing 👍